recipe.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+
+<h1>The Perfect PB&J</h1>
+<p>
+    <img src="https://images.media-allrecipes.com/userphotos/720x405/6657.jpg"/>
```

Comment:
img tag should have an alt attribute that specifies an alternate text for an image